### PR TITLE
chore: setup 1.2.0

### PR DESCRIPTION
**Pull request by BrunoQuaresma**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
 ## 1.1.0
 
 * Highlight FQL expression and run selected FQL query. [jfloresremar](https://github.com/jfloresremar)
 
 ## 1.0.0
 
-* Browse databases, indexes, collections, documents and functions.
-* Run queries.
\ No newline at end of file
+* Browse databases, indexes, collections, documents and functions
```
Comment (n400) [inline CHANGELOG.md:15]: ...nts, and functions.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,16 @@
 # Change Log
 
+## 1.2.0
+
+* Add commands to upload, merge, override GraphQL schema. [nksaraf](https://github.com/nksaraf)
```
Comment (n400) [inline CHANGELOG.md:5]: ...merge, and override Graph...

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,10 +1,16 @@
...
+* Add flexible auth via a .faunarc config file. [gahabeen](https://github.com/gahabeen)
+* Add `Documents()` on query runner
```
Comment (n400) [inline CHANGELOG.md:7]: ...runner.

**Comment by danieltodonnell** [comment]
@BrunoQuaresma I added Summer's comments.  Should be good to squash now.

**Pull request merged**